pymc/tests/test_step.py

```diff
@@ -117,11 +117,11 @@ def test_step_continuous(self):
         for step in steps:
             idata = sample(
                 0,
-                tune=8000,
+                tune=12000,
```

Comment:
Let's not include this change in this PR. This test is quite lengthy already, and it's better to let it fail now and then to motivate someone to properly fix it, instead of monkey-patching it every time it fails in unrelated PRs. 

Otherwise your PR is ready to merge

Reply:
Alright, I dropped those commits for now.